robot_smach_states/src/robot_smach_states/console.py

```diff
@@ -42,7 +42,7 @@ def load_robot(robot_name, parts=None):
     loaded = []
     for name in modules:
         try:
-            print bcolors.OKBLUE + '\tloading %s' % name + bcolors.ENDC
+            print(bcolors.OKBLUE + '\tloading %s' % name + bcolors.ENDC)
```

Comment:
I'm missing the `from __future__ import print_function`. Without this, this code doesn't make any sense.

Reply:
I am not sure this is required anymore. I have used `print(X)' many times in python 2.7 without importing the python 3 print function. Maybe it is imported automatically. I don't know what is happening under the hood.

Reply:
Try the following code
```py
print(3),
```
In python 2 it works. In python 3 it doesn't and it should not. The comma is wrong. The `from __future__ import print_function` call switches the old behavior to the new.